### ISSUE TITLE
[QUESTION] Setting options via query string

### ISSUE THREAD

**Issue by Rudi9719** [benbusby/whoogle-search#144]
Is it possible to set config options (like Language, dark mode or region) via the search query?

/search?lang=English&dark=True&q=Test&ie=UTF-8

**Comment by benbusby** [comment]
Currently, no, but I'm open to the idea. I don't think I'd want to support the full suite of configuration settings as search params, but things like dark mode/language/region should be fine. What's the use case for something like this? I can see it being useful for public instances of whoogle, but I'm curious what you had in mind.

**Comment by Rudi9719** [comment]
https://whoogle.sdf.org - I'm constantly changing the language/region back to English, as SDF is international and has multiple people using your amazing project! I've overriden the default search bar in Chrome, and KDE to use SDF's public whoogle instance

**Comment by benbusby** [comment]
Added in 72cbc342af313dba9f332704dbabf58dc7bcbcdf. Once this is merged into a release build, I can reach out to SDF to see if they'd mind updating to the version with this fix (or they can cherry pick that commit into their current instance, if they read this). 

In your use case, I think setting the query as `/search?q=%s&dark=1&lang_interface=lang_en&lang_interface=lang_en` should work. I'll do a more in depth wiki write up once this is merged into master. Thanks for the recommendation!

**Comment by Rudi9719** [comment]
Sorry to necro, but I wanted to reach out and say THANK YOU SO MUCH!!!

**Comment by benbusby** [comment]
No problem! I'm happy to help.
